.circleci/config.yml

```diff
@@ -245,3 +260,126 @@ jobs:
           name: run go build
           command: GOPATH=$(pwd) GOROOT=$(pwd)/go ./go/bin/go build ./go/src/...
           working_directory: /tmp/u-root-test
+  extra_files:
+    docker:
+      - image: circleci/golang:latest
+    working_directory: /go/src/github.com/u-root/u-root
+    environment:
+      - CGO_ENABLED: 0
+    steps:
+      - checkout
+      - run:
+          name: Install cpio
+          command: sudo apt-get install cpio
+      - run:
+          name: Build u-root
+          environment:
+            - GOARCH: amd64
+          command: go build u-root.go
+      - run:
+          name: Build ramfs
+          command: ./u-root -build=bb --tmpdir=/tmp/u-root -files /bin/bash
+      - run:
+          name: make test directory
+          command: mkdir /tmp/u-root-test
+      - run:
+          name: extract archive
+          command: cpio -i -d --no-absolute-filenames "bin/bash" < /tmp/initramfs.linux_amd64.cpio
+          working_directory: /tmp/u-root-test
+  extra_files_multiple_files:
+    docker:
+      - image: circleci/golang:latest
+    working_directory: /go/src/github.com/u-root/u-root
+    environment:
+      - CGO_ENABLED: 0
+    steps:
+      - checkout
+      - run:
+          name: Install cpio
+          command: sudo apt-get install cpio
+      - run:
+          name: Build u-root
+          environment:
+            - GOARCH: amd64
+          command: go build u-root.go
+      - run:
+          name: Build ramfs
+          command: ./u-root -build=bb --tmpdir=/tmp/u-root -files /bin/bash -files /bin/ls
+      - run:
+          name: make test directory
+          command: mkdir /tmp/u-root-test
+      - run:
+          name: extract archive
+          command: cpio -i -d --no-absolute-filenames "bin/bash bin/ls" < /tmp/initramfs.linux_amd64.cpio
+          working_directory: /tmp/u-root-test
+  extra_files_comma_syntax:
+    docker:
+      - image: circleci/golang:latest
+    working_directory: /go/src/github.com/u-root/u-root
+    environment:
+      - CGO_ENABLED: 0
+    steps:
+      - checkout
+      - run:
+          name: Install cpio
+          command: sudo apt-get install cpio
+      - run:
+          name: Build u-root
+          environment:
+            - GOARCH: amd64
+          command: go build u-root.go
+      - run:
+          name: Build ramfs
+          command: ./u-root -build=bb --tmpdir=/tmp/u-root -files /bin/bash:bin/bush
```

Comment:
bin/bush?

Reply:
yes, to confirm that file with a different name from the source will work too